lib/diagnostic.rb

```diff
@@ -7,36 +7,41 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    longest = list.map(&:length).max
+    p longest
```

Comment:
This is close but when you use `max` you can pass it a block so it knows how to determine the max. Or you can use `max_by` like this:

`list.max_by(&:length)`